File: online-judge-solutions/Codeforces/1721A.py
#https://codeforces.com/problemset/problem/1721/A
#Lang		:	Python 3.8
#Time		:	61 ms
#Memory         :	0 KB
for _ in range(int(input())):
    a,b,s=input(),input(),set()
    s.update(a+b);
    print(len(s)-1)

File: online-judge-solutions/Codeforces/1721A.cpp
//https://codeforces.com/problemset/problem/1721/A
//Lang		:	GNU C++14
//Time		:	0 ms
//Memory	:	0 KB
#include<iostream>
#include<set>
using namespace std;
int main(){
	int t;
	char x;
	cin>>t;
	while(t--){
		int a=4;
		set<int>s;
		while(a--){cin>>x;s.insert(x);}
		cout<<s.size()-1<<endl;
	}
	return 0;
}
